ProteoformSuiteGUI/ExperimentTheoreticalComparison.cs

```diff
@@ -385,6 +385,7 @@ private void nUD_PeakCountMinThreshold_ValueChanged(object sender, EventArgs e)
                 Parallel.ForEach(SaveState.lollipop.et_peaks, p =>
                 {
                     p.Accepted = p.peak_relation_group_count >= SaveState.lollipop.min_peak_count_et;
+                    p.count_nearby_decoys(SaveState.lollipop.ed_relations.Values.SelectMany(v => v).ToList());
```

Comment:
Can you make a Dictionary<string, List<ProteoformRelation>> of decoy relations in DeltaMassPeak, and then just enter them into the peak after the first time counting?